RandLAPACK/drivers/rl_cqrrpt.hh

```diff
@@ -37,7 +37,19 @@ template <typename T>
 class CQRRPT : public CQRRPTalg<T> {
     public:
 
-        // Constructor
+        /// The algorithm allows for choosing how QRCP is emplemented: either thropught LAPACK's GEQP3
+        /// or through a custom HQRRP function. This decision is controlled through 'no_hqrrp' parameter,
+        /// which defaults to 1.
+        ///
+        /// The algorithm allows for choosing the rank estimation schemeL either naively, through looking at the
+        /// diagonal entries of an R-factor from QRCP or via finding the smallest k such that ||A[k:, k:]||_F <= tau_trunk * ||A||_F.
+        /// This decision is controlled through 'naive_rank_estimate' parameter, which defaults to 1.
+        ///
+        /// The algorithm optionally times all of its subcomponents through a user-defined 'verbosity' parameter.
+        ///
+        /// The algorithm optionally computes a condition number of a preconditioned matrix A through a 'cond_check'
+        /// parameter, which defaults to 0. This requires extra n * (m + 1) * sizeof(T) bytes of space, which will be 
+        /// internally allocated by a utility routine. 
```

Comment:
Clarify: this is the $\ell_2$ condition number, right? If so, then would be good to mention that you compute this by computing all singular values with an appropriate LAPACK function.

Reply:
Fixed.